src/mono/browser/build/BrowserWasmApp.targets

```diff
@@ -459,9 +458,10 @@
     </PropertyGroup>
 
     <ItemGroup>
-      <_MonoRuntimeComponentDontLink Include="wasm-bundled-timezones.a" Condition="'$(InvariantTimezone)' == 'true'"/>
       <_MonoRuntimeComponentDontLink Include="libmono-component-diagnostics_tracing-static.a" Condition="'$(EnableDiagnostics)' != 'true'" />
       <_MonoRuntimeComponentDontLink Include="libmono-component-diagnostics_tracing-stub-static.lib" Condition="'$(EnableDiagnostics)' == 'true'" />
+      <_MonoRuntimeComponentDontLink Include="libSystem.Native.TimeZoneData.Disabled.a" Condition="'$(InvariantTimezone)' != 'true'" />
```

Comment:
let's call this `libSystem.Native.TimeZoneData.Invariant.a` ?

Reply:
Done.